docker-compose-mitre.yml

```diff
@@ -0,0 +1,38 @@
+version: "3"
+
+services:
+  inferno:
+    image: tacoma/deqm-test-client:0.1.0
+    ports:
+      - "4567:4567"
+  cqf_ruler:
+    image: artifacts.mitre.org:8200/cqf-ruler-preloaded:0.1.3.no-vs
```

Comment:
why not use the dockerhub image here?

Reply:
Good point, we could since this is the `no-vs` one--or we could point to the one with valuesets for the `docker-compose-mitre.yml`.  Which do you prefer?

Reply:
dockerhub. until we have a reason to use ValueSets

Reply:
updated to use dockerhub after discussion in slack